src/signals/incident/components/form/MapSelectors/Asset/Selector/WfsLayer/StatusLayer/StatusLayer.tsx

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2021 Gemeente Amsterdam
+import L from 'leaflet'
+import './style.css'
+
+import type { FC } from 'react'
+import type {
+  Feature,
+  FeatureType,
+} from 'signals/incident/components/form/MapSelectors/types'
+import { Marker } from '@amsterdam/arm-core'
+import { featureToCoordinates } from 'shared/services/map-location'
+import type { Geometrie } from 'types/incident'
+import reportedIconUrl from 'shared/images/icon-reported-marker.svg?url'
+import checkedIconUrl from 'shared/images/icon-checked-marker.svg?url'
+
+const STATUS_CLASS_MODIFIER = 'marker-status'
+
+export interface StatusLayerProps {
+  statusFeatures: Feature[]
+  reportedFeatureType: FeatureType
+  checkedFeatureType: FeatureType
+}
+
+const StatusLayer: FC<StatusLayerProps> = ({
+  statusFeatures,
+  reportedFeatureType,
+  checkedFeatureType,
+}) => {
+  const getMarker = (feat: any, index: number) => {
+    const feature = feat as Feature
+    const latLng = featureToCoordinates(feature?.geometry as Geometrie)
+
+    if (!feature || !reportedFeatureType) return
```

Comment:
`reportedFeatureType` Is a required prop; is it necessary to check for its presence?

Reply:
removed that also